pkg/controller/controller.go

```diff
@@ -22,11 +22,6 @@ import (
 
 var startTime time.Time
 
-const (
-	controllerStartMsg = "...and now my watch begins! :crossed_swords:"
-	controllerStopMsg  = "my watch has ended!"
-)
-
```

Comment:
Don't remove constants. Instead, use formatting methods like fmt.Sprintf() for string manipulations

Reply:
Added constants and used Sprintf